src/main/scala/io/kf/etl/processors/featurecentric/transform/FeatureCentricTransformer.scala

```diff
@@ -213,14 +230,19 @@ object FeatureCentricTransformer {
 
   private def genomicFile_ES_to_FileCentric(
                                              genomicFile: GenomicFile_ES,
-                                             participants: Seq[Participant_ES]
+                                             participants: Seq[Participant_ES],
+                                             mapDataCategoryTypes: Map[String, Seq[String]] = Map.empty[String, Seq[String]]
                                            ): FileCentric_ES = {
     FileCentric_ES(
       acl = genomicFile.acl,
       availability = genomicFile.availability,
       access_urls = genomicFile.access_urls,
       controlled_access = genomicFile.controlled_access,
       data_type = genomicFile.data_type,
+      data_category = genomicFile.data_type match {
```

Comment:
you can use genomicFile.data_type.map(d -> calculateDataCategory(Seq(d), mapDataCategoryTypes))
if you don't want to do anything with the None

or a flatMap if `calculateDataCategory` returns an Option